src/policies/conditions.ts

```diff
@@ -82,6 +82,21 @@ export class ConditionSet {
 }
 
 export class Condition {
+  // TODO: Shared types, move them somewhere?
```

Comment:
Does the recent Rust work expose these?

Reply:
No *new* types are introduced in nucypher core 0.3.0.